docs/guides/moodleapp/remote-themes.md

```diff
@@ -0,0 +1,401 @@
+---
+title: Moodle App Remote Themes
```

Comment:
Same comment as for the other page. Apart from that, I think it would be better to have "Remote themes" (themes without a capital letter).

Reply:
Ok, removed the capital but waiting for #74

Reply:
Solved and done